modules/ROOT/pages/con_pipeline-git-resources.adoc

```diff
@@ -0,0 +1,84 @@
+[]
+= Git Resource
+
+Git resource represents a [git](https://git-scm.com/) repository, that contains
```

Comment:
@joaedwar: This link address is not properly formed, I think. I can see git in square brackets ([git]) in rendered text. You need the link statement first, followed by the web address and the link text in square brackets:
link:https://git-scm.com[git]